helpers/labgrid-raw-interface

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
+#
+# Wrapper script to be deployed on machines whose network interfaces should be
+# controllable via the RawNetworkInterfaceDriver. A /etc/labgrid/labgrid.ini
+# can deny access to network interfaces. See below.
+#
+# This is intended to be used via sudo. For example, add via visudo:
+# %developers ALL = NOPASSWD: /usr/local/sbin/labgrid-raw-interface
+
+import argparse
+import os
+import sys
+import configparser
+import warnings
+
+
+def main(program, ifname, capfile, count):
+    if not ifname:
+        raise ValueError("Empty interface name.")
+    if any((c == "/" or c.isspace()) for c in ifname):
+        raise ValueError(f"Interface name '{ifname}' contains invalid characters.")
+    if len(ifname) > 16:
+        raise ValueError(f"Interface name '{ifname}' is too long.")
+
+    config = configparser.ConfigParser()
```

Comment:
I'm not convinced using two different config file formats is a good idea. Can't we use YAML here as well?

Reply:
Discussed this with @Emantor: We can use a /etc/labgrid/helpers.yaml with a structure like this:
``` yaml
raw-interface:
  denied-interfaces:
    - eth0
```

Reply:
Done.